flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/internal/options/JdbcOptions.java

```diff
@@ -34,7 +34,7 @@
 
 	private static final long serialVersionUID = 1L;
 
-	public static final int CONNECTION_CHECK_TIMEOUT_SECONDS = 60;
+	public static  int connectionCheckTimeoutSeconds = 60;
```

Comment:
This shouldn't be a global variable. This should be a `private final` variable and please also add this into `hashcode()` and `equals()`.